### PR TITLE
cdc: fix panic caused by insert

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### What problem does this PR solve?

Problem Summary:

If an insert operation has an old value, tikv would be panic by an assertion.

### What is changed and how it works?

Delete the assertion.

What's Changed:

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test

### Release note 
* N/A

### CODE REVIEW

**Comment by liuzix** [comment]
LGTM

File: components/cdc/tests/integrations/test_cdc.rs [hunk 1]
```diff
@@ -817,7 +817,16 @@ fn test_old_value_basic() {
...
+    suite.must_kv_pessimistic_prewrite(1, vec![m7], k1.clone(), ts9, ts12);
+    let ts13 = block_on(suite.cluster.pd_client.get_tso()).unwrap();
+    suite.must_kv_commit(1, vec![k1.clone()], ts9, ts13);
+    // Insert value again
+    let mut m8 = Mutation::default();
+    m8.set_op(Op::Insert);
+    m8.key = k1.clone();
+    m8.value = b"v1".to_vec();
+    let ts14 = block_on(suite.cluster.pd_client.get_tso()).unwrap();
+    suite.must_kv_prewrite(1, vec![m8], k1.clone(), ts14);
```
Comment (amyangfei) [inline components/cdc/tests/integrations/test_cdc.rs:829]:
```suggestion
    suite.must_kv_prewrite(1, vec![m8], k1, ts14);
```

**Comment by amyangfei** [comment]
rest lgtm

**Pull request merged**